sql/src/test/java/io/crate/analyze/CopyAnalyzerTest.java

```diff
@@ -44,12 +44,8 @@
 import static com.carrotsearch.randomizedtesting.RandomizedTest.$;
 import static io.crate.analyze.TableDefinitions.TEST_PARTITIONED_TABLE_IDENT;
 import static io.crate.analyze.TableDefinitions.USER_TABLE_IDENT;
-import static io.crate.testing.SymbolMatchers.isFunction;
-import static io.crate.testing.SymbolMatchers.isLiteral;
-import static io.crate.testing.SymbolMatchers.isReference;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.instanceOf;
-import static org.hamcrest.Matchers.is;
+import static io.crate.testing.SymbolMatchers.*;
+import static org.hamcrest.Matchers.*;
```

Comment:
We don't use star imports (but currently only enforce this in the main scope, not test).

Reply:
I think that was intellij being a bit smart... Updated my preferences so it won't do that again